windows/rngallery/packages.lock.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Why are we just now starting to get a packages.lock.json? And specifically in the windows/rngallery folder?
(I've been seeing the same thing locally, FWIW, but I don't understand why)

Reply:
I'm not sure, I've been deleting it every time I make a PR because it seems unnecessary but haven't done it to this test branch. If everyone is getting it, I'm fine to include it in the PR